packages/react-board/src/setup-stage.tsx

```diff
@@ -0,0 +1,179 @@
+import type {
+    CanvasStyles,
+    ICanvasEnvironmentProps,
+    IPreviewEnvironmentPropsBase,
+    IWindowEnvironmentProps,
+} from '@wixc3/board-core';
+import type { BoardSetupStageFunction } from './types';
+
+export const defaultWindowStyles = {
+    width: 1024,
+    height: 640,
+} as const;
+
+export const defaultCanvasStyles: CanvasStyles = {
+    width: 'fit-content',
+    height: 'fit-content',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginBottom: 'auto',
+    marginTop: 'auto',
+    paddingLeft: '0px',
+    paddingRight: '0px',
+    paddingBottom: '0px',
+    paddingTop: '0px',
+} as const;
+
+export const defaultEnvironmentProperties: IPreviewEnvironmentPropsBase = {
+    windowWidth: defaultWindowStyles.width,
+    windowHeight: defaultWindowStyles.height,
+    canvasMargin: {},
+    canvasPadding: {},
+};
+
+export const defaultWindowProperties: IWindowEnvironmentProps = {
+    height: defaultWindowStyles.height,
+    width: defaultWindowStyles.width,
+};
+
+export const defaultCanvasProperties: ICanvasEnvironmentProps = {};
+
+const applyStylesToWindow = (windowStyles: IWindowEnvironmentProps = {}, previousProps: IWindowEnvironmentProps) => {
+    // we revert the changes to previous values when running cleanup
+    previousProps.height = previousProps.height ? window.outerHeight : defaultWindowStyles.height;
+    previousProps.width = previousProps.width ? window.outerWidth : defaultWindowStyles.width;
+
+    window.resizeTo(windowStyles.width || previousProps.width, windowStyles.width || previousProps.width);
```

Comment:
both parameters are `width`
```suggestion
const width = windowStyles.width || previousProps.width;
const height = windowStyles.height || previousProps.height; 
    window.resizeTo(width, height);
```